src/utils/getFilteredPeople.ts

```diff
@@ -0,0 +1,48 @@
+import { Person } from '../types';
+
+export const getFilteredPeople = (
+  people: Person[],
+  query: string | null,
+  sex: string | null,
+  centuries: string[],
+) => {
+  let filteredPeople = [...people];
+
+  if (query) {
+    filteredPeople = filteredPeople.filter(person => {
+      return person.name.toLowerCase()
+        .includes(query.toLowerCase())
```

Comment:
let's prepare the `query` before calling `filter` method
for now you transform it each time on each callback call